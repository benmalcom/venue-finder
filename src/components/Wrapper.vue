<template>
    <!-- wrapper template -->
    <div class="wrapper flex-grow">
        <slot></slot>
    </div>
</template>


<script>
	export default {
		name: 'Wrapper'
	}
</script>

<style scoped>

</style>
