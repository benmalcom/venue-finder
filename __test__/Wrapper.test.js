import { shallow } from 'vue-test-utils'
import Wrapper from '../src/components/Wrapper.vue';

describe('Component: Wrapper.vue', () => {
	let comp;

	beforeEach(() => {
		comp = shallow(Wrapper, { // Create a shallow instance of the component
		});
	});

	it('Has the expected html structure', () => {
		expect(comp.element).toMatchSnapshot();
	});

	it('Should contain a div element with class wrapper', () => {
		const div = comp.find('div.wrapper');
		expect(div).toBeDefined();
	});

	it('Should contain a slot inside a div child', () => {
		const slot = comp.find('div.wrapper slot');
		expect(slot).toBeDefined();
	});
});
