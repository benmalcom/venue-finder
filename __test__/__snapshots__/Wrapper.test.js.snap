// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Wrapper.vue Has the expected html structure 1`] = `
<div
  class="wrapper"
/>
`;
